addons/loyalty/report/loyalty_report_templates.xml

```diff
@@ -75,36 +75,34 @@
 
     <template id="gift_card_report">
         <t t-call="web.html_container">
-            <t t-foreach="docs" t-as="o">
```

Comment:
why did you need to remove that?

Reply:
because this template is still called in a `foreach` loop for every gift card in `gift_card_report_i18n`, this ends up creating a bug where if you have 2 gift cards, it would print 4 (each card twice), if you have 3, it would print 9, etc